jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -147,7 +147,14 @@ public static JibBuildRunner createJibBuildRunnerForRegistryImage(
         inferredAuthProvider,
         rawConfiguration.getToCredHelper().orElse(null));
 
+    boolean isAlwaysCacheBaseImage =
+        Boolean.getBoolean(
+            rawConfiguration
+                .getProperty(PropertyNames.ALWAYS_CACHE_BASE_IMAGE)
+                .orElse(Boolean.FALSE.toString()));
     Containerizer containerizer = Containerizer.to(targetImage);
+    containerizer.setAlwaysCacheBaseImage(isAlwaysCacheBaseImage);
```

Comment:
You can chain these: `Containerizer.to(...).setAlways...`

Reply:
Done.